README.md

```diff
@@ -193,7 +193,7 @@ on the subdirectories of that folder.
 
 If a PR is altering the CLI output, files inside those directories might
 need to be manually updated. This often happens when options are
-added, removed or altered. Keep in mind that the version string line
+added, removed, or altered. Keep in mind that the version string line
```

Comment:
I don't think this comma is required since it is followed by `or`. I am not a native English speaker but I am quite sure commas are not required if followed by a conjunction.

Reply:
It depends on how many items you use. If you have more than two items, you need a comma. See https://www.grammarly.com/blog/comma-before-or/ for more information.

Reply:
Based on the link, it seems both forms are acceptable. I will merge the PR as is then.